src/test/java/connectify/logic/parser/ConnectifyParserTest.java

```diff
@@ -150,6 +151,22 @@ public void parseCommand_findCompanies() throws Exception {
         assertEquals(new FindCompaniesCommand(new CompanyNameContainsKeywordsPredicate(keywords)), command);
     }
 
+    @Test
+    public void parseCommand_shareCompany() throws Exception {
+        ShareCompanyCommand command = (ShareCompanyCommand) parser.parseCommand(
+                ShareCompanyCommand.COMMAND_WORD + " " + INDEX_FIRST_COMPANY.getOneBased());
```

Comment:
just curious why did you use "getOneBased"? instead of just `INDEX_FIRST_COMPANY`